<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FabProcess</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="cmdGestor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>232</y>
      <width>200</width>
      <height>116</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Configuracion de S.O</string>
    </property>
    <property name="icon">
     <iconset resource="static_files.qrc">
      <normaloff>:/process_status.png</normaloff>:/process_status.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdSalir">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>464</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>252</red>
          <green>1</green>
          <blue>7</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(252, 1, 7);
font: 75 14pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Documents/Imagenes_Proyecto/new_workorder-512.png</normaloff>../Documents/Imagenes_Proyecto/new_workorder-512.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdNewProcess">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>116</y>
      <width>200</width>
      <height>116</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(144, 180, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nuevo Proceso</string>
    </property>
    <property name="icon">
     <iconset resource="static_files.qrc">
      <normaloff>:/process_add.png</normaloff>:/process_add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>44</width>
      <height>44</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Shift+N</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="cmdLogo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>116</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(144, 180, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="static_files.qrc">
      <normaloff>:/FabProccess.png</normaloff>:/FabProccess.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>116</height>
     </size>
    </property>
    <property name="shortcut">
     <string>Shift+N</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QFrame" name="panelNew">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>600</width>
      <height>525</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>361</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Nuevo Proceso</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>543</x>
       <y>5</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="static_files.qrc">:/process_add.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>600</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtNombre">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Nombre:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Tiempo:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Memoria:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="upMemoria">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QSpinBox" name="upTiempo">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdAgregar">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>151</width>
       <height>116</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="static_files.qrc">
       <normaloff>:/add.png</normaloff>:/add.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="tbProcesos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>521</width>
       <height>192</height>
      </rect>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column/>
     <column/>
     <column/>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>500</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>**Doble click para eliminar un proceso.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdSubir">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>300</y>
       <width>70</width>
       <height>90</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(15, 128, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="static_files.qrc">
       <normaloff>:/swipe-up.png</normaloff>:/swipe-up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="cmdBajar">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>402</y>
       <width>70</width>
       <height>90</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(15, 128, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="static_files.qrc">
       <normaloff>:/down-side.png</normaloff>:/down-side.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <zorder>line</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>txtNombre</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>upMemoria</zorder>
    <zorder>upTiempo</zorder>
    <zorder>cmdAgregar</zorder>
    <zorder>tbProcesos</zorder>
    <zorder>label_6</zorder>
    <zorder>cmdSubir</zorder>
    <zorder>cmdBajar</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="static_files.qrc"/>
 </resources>
 <connections/>
</ui>
